nav2_system_tests/src/costmap_filters/keepout_params.yaml

```diff
@@ -119,6 +119,7 @@ controller_server:
       plugin: "dwb_core::DWBLocalPlanner"
       debug_trajectory_details: True
       prune_distance: 1.0
+      shorten_distance: 2.0
```

Comment:
This param needs to be added to our default param files in the nav2_bringup/params directory

Reply:
Ok, note that `prune_plan`, `prune_distance`, and `shorten_transformed_plan` are currently not set in those configuration files. Do you confirm that I should add this new parameter?

Reply:
Why was it added here?

Reply:
Because `prune_distance` is set here to a non-default value. So should be `forward_prune_distance` to keep the same behavior. Am I wrong?

Reply:
Looking at the diff, it doesn't look like there was ever a line in that file for `prune_distance`, it appears on my reading of this that you added a parameter that wasn't previously there.

Reply:
Ok as you wish. Then the default value of `forward_prune_distance` will be used and it won't match the value of `prune_dist`.